### ISSUE TITLE
Add Fuzzing test suites in Litmus

### ISSUE THREAD

**Issue by Saranya-jena** [litmuschaos/litmus#4548]
**What happened**:

**What you expected to happen**:

**Where can this issue be corrected? (optional)**

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Comment by milinddethe15** [comment]
Isn't this issue included for Gsoc project (#4425)?

**Comment by Baalekshan** [comment]
Hi @Saranya-jena , could you provide more context on this issue.

**Comment by anshikavashistha** [comment]
@Saranya-jena For adding fuzzing test suites in litmus ,we may add AFL, libFuzzer and some more tools that may be compatible with litmus.I don't much about the compatibility. Please guide me

**Comment by Baalekshan** [comment]
@Saranya-jena I will take this.

**Comment by Baalekshan** [comment]
@prithvi1307 @Saranya-jena I'll work on this

**Comment by Akash-Singh04** [comment]
Hey @Baalekshan Are you still working on this?

**Comment by Baalekshan** [comment]
@Akash-Singh04  Yeah, I do but If you want to work on this I think you can also add test suites if you want, just let me know what part of the codebase you are covering so both might not be working on the same thing.

**Comment by Akash-Singh04** [comment]
@Baalekshan No worries I'll take up another issue, All the best!

**Comment by sagnik3788** [comment]
@Baalekshan are you still working on this ?

**Comment by Baalekshan** [comment]
No, @sagnik3788 feel free to take it !

**Comment by vividwhisp** [comment]
@Saranya-jena @Baalekshan can i contribute as i am a new contributer and new to open source but if you have been assigned then it sok
